php/lang/security/md5-loose-equality.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
+- id: md5-loose-equality
+  patterns:
+  - pattern-either:
+    - pattern: $X == $FUNC(...)
+    - pattern: $FUNC(...) == $X
+    - pattern: $FUNC(...) == $FUNC(...)
+  - metavariable-regex:
+      metavariable: $FUNC
+      regex: md5|md5_file
+  message: |
+    Make sure comparisons involving md5 values are strict to avoid type juggling issues
```

Comment:
Does "strict" mean using the `===` operator? If so, perhaps the message could be made more actionable

Reply:
It does 🙂 would you like me to update the message?